src/renderer/components/configure-recording-set-page/set-recording-list-configuration.tsx

```diff
@@ -5,7 +5,7 @@ import Row from 'react-bootstrap/Row';
 import { useTranslation } from 'react-i18next';
 
 import { Consumer } from '../../types';
-import { filename, openFilePicker } from '../../utils';
```

Comment:
Why is this being passed in?

Reply:
I refactored the code to extract the function out because I was unsure how to mock it when it was simply being used inside the component

Reply:
Can you mock the `utils` module?